src/PostgreSql/docs/Update-AzPostgreSqlFlexibleServer.md

```diff
@@ -206,21 +205,6 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
-### -ReplicationRole
```

Comment:
Is this a breaking change?

Reply:
checked by https://github.com/Azure/azure-powershell/pull/20631 no breaking change